src/pysdmx/util/handlers.py

```diff
@@ -0,0 +1,59 @@
+"""Handlers file provide functions to make the code more readable."""
+
+from typing import Any
+
+
+# import pandas as pd
+
+
+def split_unique_id(obj_: str) -> tuple[str, str, str]:
+    """Split unique id.
+
+    Args:
+        obj_: String with the name of the object.
+
+    Returns:
+        agencyID, id and version.
+    """
+    data = obj_.split(":", 1)
+    agencyID = data[0]
+    data = data[1].split("(", 1)
+    id = data[0]
+    version = data[1].split(")", 1)[0]
+
+    return agencyID, id, version
+
+
+def split_from_urn(obj_: str, split_id: bool = True) -> Any:
+    """Split from urn.
+
+    Args:
+        obj_: String with the name of the object.
+        split_id: True splits the id.
+
+    Returns:
+        full id.
+    """
+    full_id = obj_.split("=", 1)[1]
+    return split_unique_id(full_id)
```

Comment:
These 2 functions sound similar to me to the parse_urn function in this file: https://github.com/bis-med-it/pysdmx/blob/develop/src/pysdmx/util/__init__.py?

Reply:
I will recheck it, these are functions that come directly from sdmxthon so probably I missed something. The parse_urn function worked fine for metadata so it should also work here in theory.

Reply:
Solved in latest commit and used parse_urn.